web/html/xui/settings.html

```diff
@@ -394,13 +399,34 @@ <h2 style="color: inherit; font-weight: bold; font-size: 18px; padding: 10px 20p
             },
             async resetXrayConfigToDefault() {
                 this.loading(true);
-                const msg = await HttpUtil.get("/xui/setting/getDefaultJsonConfig");
+                const msg = await HttpUtil.get("/panel/setting/getDefaultJsonConfig");
                 this.loading(false);
                 if (msg.success) {
                     this.templateSettings = JSON.parse(JSON.stringify(msg.obj, null, 2));
                     this.saveBtnDisable = true;
                 }
             },
+            checkForGeosites() {
+                const domainsToCheck = [
+                    {
+                        query: "category-ru-gov",
+                        key: "this.settingsData.domains.ru",
+                        data: [
+                            "geosite:category-ru-gov",
+                            "regexp:.*\\.ru$"
+                        ]
+                    },
+                ];
+                this.loading(true);
+                domainsToCheck.forEach(async (dd) => {
+                    const msg = await HttpUtil.get(`/panel/setting/searchDatafiles?query=${dd.query}`);
+                    if (msg.success && msg.obj) {
+                        [dd.key] = dd.data;
+                        console.log([dd.key])
+                    }
+                })
+                this.loading(false);
+            },
```

Comment:
remove

Reply:
i'll remove it myself thank you for your great work
 @hamid-gh98

Reply:
Your welcome.
You can also remove the API route I added for this part, or I can do it if you want.

Reply:
By the way, I didn't change the path of the API (`/xui/API/inbounds`) because it may break some apps that using the api of panel. However, feel free to change it to (`/panel/api/inbounds`) if you find it to be better, or if you prefer, I can make the change myself. Just let me know your preference.

@MHSanaei

Reply:
I guess no need to change that maybe later